packages/synapse-react-client/src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -32,9 +36,10 @@ type State = {
 export default function DownloadDetails(props: DownloadDetailsProps) {
   const { numFiles, numBytes } = props
   const { accessToken } = useSynapseContext()
+  const hasNumBytes = !!numBytes
 
   const [state, setState] = useState<State>({
-    isLoading: !!numBytes, // figure out the estimated download time iff we were given a byte count
+    isLoading: hasNumBytes, // figure out the estimated download time iff we were given a byte count
```

Comment:
numBytes was evaluated a number in the conditional component expression, as described [here](https://stackoverflow.com/questions/53048037/react-showing-0-instead-of-nothing-with-short-circuit-conditional-component). Create an explicit boolean to track whether numByte exists so that 00 does not appear when there is no size/time.